learn/tanssi/staking.md

```diff
@@ -0,0 +1,119 @@
+
+All rewards (for all pools) are stored in a protocol-owned account. Nevertheless, the protocol internally keeps track of the actual native tokens held by each pool. The core difference between staking through the Manual or Auto-Compound Rewards Pools is how rewards are distributed. In the Manual Rewards Pool, users have to claim any staking rewards they've accumulated manually. In contrast, in the Auto-Compound Rewards Pool, the rewards are automatically re-staked at each Tanssi block, where the protocol announces the block producer for each block production assignment, either in Tanssi or an Appchain.
+
+The delegate and undelegate operations need to be sent by the delegator itself. They signal the intent of the action to be taken and ask the protocol to perform the necessary checks to allow the delegator to delegate or undelegate. Consequently, these actions can be executed only after a certain number of sessions, but anyone in the network can perform this second operation through the execute pending operation transaction.
+
+The following diagram summarizes the high-level flow of a delegator delegating and undelegating tokens to a block producer. User actions are highlighted in cyan, while different pools are highlighted in coral.
+
+![High-level overview of Tanssi Staking mechanics](/images/learn/tanssi/staking/staking-1.webp)
+
+## Staking Paremeters {: #staking-parameters }
+
+=== "Dancebox"
+    |             Variable             |                                                                         Value                                                                         |
+    |:--------------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------:|
+    |          Joining Delay          |                        {{ networks.dancebox.staking.joining_delay_blocks }} blocks ({{ networks.dancebox.staking.joining_delay_hours }} hours)                        |
+    |          Leaving Delay          |                        {{ networks.dancebox.staking.leaving_delay_blocks }} blocks ({{ networks.dancebox.staking.leaving_delay_hours }} hours)                        |
+
+## Staking Pools {: #staking-pools}
+
+The following section goes through each of the liquidity pools that represent a step throughout the staking process.
+
+### Joining Pool {: $joining-pool}
+
+When a user first delegates to start the staking process, it must state what staking rewards mechanism it wants: manual or auto-compound rewards (each being a separate pool). Once the joining transaction is executed, the user automatically enters the Joining Pool and is given shares of that pool directly correlated to the number of tokens being staked. This pool offers stability to the current set of block producers by providing a delay between a delegator staking and receiving rewards. The delay is set to at least one entire session.
```

Comment:
In the above table, it says there is a delay of 600 blocks, which is two sessions. So it looks like this needs to be updated?

Reply:
Got catch, it should be 1200 blocks now that sessions are still set to 1 hour but we are producing blocks at 6 seconds